CraftTweaker2-API/src/main/java/crafttweaker/api/player/IPlayer.java

```diff
@@ -136,4 +136,13 @@ default IEntityItem dropItem(IItemStack item) {
         CraftTweakerAPI.logError("Default method IPlayer#dropItem(IItemStack) is not overwritten in " + getClass() + " please report to the author!");
         return null;
     }
+    @ZenGetter("sneaking")
+    @ZenMethod
+    boolean isSneaking();
+
+    @ZenMethod
+    void setCooldown(IItemStack item, float time)
+
+    @ZenMethod
+    void getCooldown(IItemStack item)
```

Comment:
Please make these default methods that have something to the effect of:
```
CraftTweakerAPI.logError("Default method IPlayer#dropItem(bool) is not overwritten in " + getClass() + " please report to the author!");
// optionally return false; for the method that returns a boolean.
```

Other mods, like ContentTweaker for example may implement IPlayer and this would break them.

Reply:
didn't we want to add new methods via expansions rather than doing this `default` way

Reply:
Yes, I forgot about that, I'll keep it in mind for the future, it should be fine here since nothing should really be extending IPlayer